packages/taquito-contracts-library/test/taquito-contracts-library.spec.ts

```diff
@@ -118,4 +146,20 @@ describe('ContractsLibrary tests', () => {
       expect(e).toBeInstanceOf(Error);
     }
   });
-});
+
+  it('Test the configure context function through use of Tezos.addExtension', () => {
```

Comment:
Unit tests should test specific functions. Testing Tezos.addExtension might be more appropriate in the taquito package

Reply:
This test adds nothing to code coverage and is mainly a repetition of an earlier test, except it adds the Tezos.addExtension,
I have removed the test.